src/renderer/pages/Project.tsx

```diff
@@ -155,10 +156,22 @@ const ProjectPage = () => {
     };
   });
 
-  // TODO: figure out return type
+  useEffect(() => {
```

Comment:
I understand for MVP but I reckon this should definitely be optimised if possible in a later PR. Please add a 'TODO: <comment-here>' comment to look into optimising this later